added magic contract usage page (#1102)
iotaledger/iota-wiki: docs/build/wasp/0.7.0/docs/guide/evm/magic.md

The Magic contract has several methods, which are categorized into specialized
interfaces: `ISCSandbox`, `ISCAccounts`, `ISCUtil` and so on.
You can access these interfaces from any Solidity contract by importing
the [ISC library](https://www.npmjs.com/package/@iota/iscmagic):

```sh
yarn add @iota/iscmagic
```

> Thereon you can import it into your contracts as such:

Reviewer: ```suggestion
You can import it into your contracts like this:
```
